doc/tech/approach.md

```diff
@@ -3,97 +3,230 @@
+
+> **Note:** Appliance-like applications that fully own its infrastructure layers (share nothing) could manage and utilize a Programmable Networking Fabric in many ways, but that is not a Cloud implementation and falls outside the use cases for these specifications.
 
 <a name="5.1.4"></a>
-### 5.1.4 Initial Logistical Steps
-
-> _**Editor's Note:** The purpose in this section is to communicate the responsibilities of the CNTT levels, as they pertain to the networking solution, and how they relate (i.e. what goes in each bucket). The initial content below needs to be expanded with examples of material representative of the scope for each bucket, and refined. It is expected this area be enhanced to help authors in various CNTT levels understand their purview, and ultimately be deleted._
-
-As with most CNTT subsystems, responsibility for Objectives, Requirements, Guidelines, etc. is divided among the CNTT constituencies, as follows:
-- **Tech:** High-level strategy and other informative writings
-  - e.g. Executive summary, considerations, intent, vision, etc.
-- **RM:** Generic modeling and abstraction
-  - e.g. Guidelines and other information applicable to all RAs
-  - Overall theory of operation
-- **RA-x:** architecture, APIs and other service level details
-- **RI-x:** networking information for the relevant lab or POD
-  - Considerations related to differences between target architecture (RM 5.1) and current RI implementation
-  - Factors specific to supporting RC
-  - Known non-conformances
-- **RC:** Details related to qualification
-  - Implications of testbed network environment vs. production environments
+### 5.1.4 Networking Reference Model
+
+The Networking Reference Model depicted in **Figure 5-1-4** is based on the ETSI NFV model enhanced with Container Virtualisation support and a strict separation of the HW Infrastructure and SW Infrastructure Layers in NFVI. It includes all above concepts and enables multiple well separated simultaneous SW Virtualisation domains allowing a mix of CaaS on Metal, CaaS on IaaS and IaaS on a shared HW infrastructure. 
+
+It is up to any deployment of  the Cloud Infrastructure to decide what Networking related objects to use, but all Reference Architectures have to be able to map into this model.
+
+<p align="center"><img src="./figures/RM_NW_Concepts_Layering-RMNW_ETSINFV-PA7.jpg" alt="Networking Reference Model based on the ETSI NFV" title="Networking Reference Model based on the ETSI NFV" width="100%"/></p>
+<p align="center"><b>Figure 5-1-4:</b> Networking Reference Model based on the ETSI NFV</p>
+
+<a name="5.1.5"></a>
+### 5.1.5 Deployment examples based on the Networking Reference Model
```

Comment:
Can we move this to a separate PR please?
The title for this is to define the scope.